packages/errors/src/errors.ts

```diff
@@ -1551,6 +1551,14 @@ export const AllCypressErrors = {
       the devServer config as a ${fmt.highlight(devServerConfigFile)} option.
     `
   },
+
+  TESTING_TYPE_NOT_CONFIGURED: (testingType: TestingType) => {
+    return errTemplate`
+      The testing type selected (${fmt.highlight(testingType)}) is not configured in your config file.
+
+      Please update it, and try running it again.
```

Comment:
Can we instruct them to run ‘cypress open’ to have the configure it?

```suggestion
      Please run ‘cypress open’ and choose your testing type to automatically update your configuration file.
```